spark/src/main/java/org/opensearch/sql/spark/client/EmrServerlessClient.java

```diff
@@ -0,0 +1,9 @@
+package org.opensearch.sql.spark.client;
+
+import org.opensearch.sql.spark.helper.FlintHelper;
+
+public interface EmrServerlessClient {
```

Comment:
can the name be more generic ?

Reply:
Will change it to spark client. Makes sense and one of the implementations is EMRServerlessClientImpl.